solutions/remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -567,6 +631,220 @@
+            "name": "[concat(parameters('iotHubName'), '/events/', variables('consumerGroupName'))]",
+            "type": "Microsoft.Devices/Iothubs/eventhubEndpoints/ConsumerGroups",
+            "dependsOn": [
+                "[concat('Microsoft.Devices/Iothubs/', parameters('iotHubName'))]"
+            ]
+        },
+        {
+            "comments": "Azure StreamAnalytics streamingjobs",
+            "apiVersion": "[variables('streamAnalyticsApiVersion')]",
+            "type": "Microsoft.StreamAnalytics/streamingjobs",
+            "name": "[parameters('streamingJobsName')]",
+            "location": "[variables('location')]",
+            "tags": {
+                "IotSuiteType": "[parameters('solutionType')]"
+            },
+            "dependsOn": [
+                "[concat('Microsoft.Storage/storageAccounts/', parameters('storageName'))]",
+                "[concat('Microsoft.Devices/Iothubs/', parameters('iotHubName'))]"
+            ],
+            "properties": {
+                "sku": {
+                    "name": "standard"
+                },
+                "eventsOutOfOrderMaxDelayInSeconds": 10,
+                "eventsOutOfOrderPolicy": "[parameters('streamingJobsEventsOutOfOrderPolicy')]",
+                "inputs": [
+                    {
+                        "name": "DeviceTelemetry",
+                        "properties": {
+                            "datasource": {
+                                "properties": {
+                                    "consumerGroupName": "[variables('consumerGroupName')]",
+                                    "iotHubNamespace": "[parameters('iotHubName')]",
+                                    "sharedAccessPolicyKey": "[listkeys(variables('iotHubKeyResource'), variables('iotHubApiVersion')).primaryKey]",
+                                    "sharedAccessPolicyName": "[variables('iotHubKeyName')]",
+                                    "endpoint": "messages/events"
+                                },
+                                "type": "Microsoft.Devices/IotHubs"
+                            },
+                            "serialization": {
+                                "properties": {
+                                    "encoding": "UTF8"
+                                },
+                                "type": "Json"
+                            },
+                            "type": "Stream"
+                        }
+                    },
+                    {
+                        "name": "DeviceGroups",
+                        "properties": {
+                            "datasource": {
+                                "properties": {
+                                    "container": "[parameters('streamingJobsInputContainerName')]",
+                                    "dateFormat": "yyyy-MM-dd",
+                                    "timeFormat": "HH-mm",
+                                    "pathPattern": "{date}/{time}/devicegroups.csv",
```

Comment:
Is the plan that we will hard-code the container name and file names in the device groups agent and telemetry agent? I think it would be better if we didn't have to do that, in case the file or container name gets changed here. We could add them as environment variables.

Reply:
you are right. To make all of these parameters be consistent with the agents, it would be better to export all of them. I can create another task for this work. For current stage it is enough for us to run it from end to end by using hard-code value, just like the TSQL time window.